doc/README.fr.md

```diff
@@ -438,12 +439,12 @@ Dessine un triangle avec les sommets (`x1`, `y1`), (`x2`, `y2`), (`x3`, `y3`) et
 Dessine les contours d’un triangle avec les sommets (`x1`, `y1`), (`x2`, `y2`), (`x3`, `y3`) et de couleur `col`
 
 - `blt(x, y, img, u, v, w, h, [colkey])`<br>
-Copie la région de taille (`w`, `h`) de (`u`, `v`) de la banque d’image `img`(0-2) à (`x`, `y`). Si une valeur négative est mise pour `w` et/ou `h`, la copie sera inversée horizontalement et/ou verticalement. Si `colkey` est spécifié, il sera traité comme une couleur transparente
+Copie la région de taille (`w`, `h`) de (`u`, `v`) de la banque d’image `img` (0-2) à (`x`, `y`). Si une valeur négative est mise pour `w` (ou `h`), la copie sera inversée horizontalement (ou verticalement). Si `colkey` est spécifiée, elle sera traitée comme une couleur transparente.
 
 <img src="images/blt_figure.png">
 
 - `bltm(x, y, tm, u, v, w, h, [colkey])`<br>
-Copy the region of size (`w`, `h`) from (`u`, `v`) of the tilemap `tm` (0-7) to (`x`, `y`). If negative value is set for `w` and/or `h`, it will reverse horizontally and/or vertically. If `colkey` is specified, treated as transparent color. The size of a tile is 8x8 pixels and is stored in a tilemap as a tuple of `(x in tile, y in tile)`.
+Copie la région de taill (`w`, `h`) de (`u`, `v`) de la tilemap `tm` (0-7) à (`x`, `y`). Si une valeur négative est mise pour `w` (ou `h`), la copie sera inversée horizontalement (ou verticalement). Si `colkey` est spécifiée, elle sera traitée comme une couleur transparente. La taille d’une tuile est 8x8 pixels et elle est storée dans une tilemap en tant que paire `(x dans tuile, y dans tuile)`.
```

Comment:
D’autres méthodes ont `(x in tile, y in tile)`, à modifier à mon sens.
(Mais pour ça il faudrait que je comprenne la doc originelle!  est-ce que «tile» est en fait «tilemap»?)

Reply:
Thank you for your confirmation.
In the latest Pyxel, basically every numbers to specify arguments for drawing functions are pixel(dot)-based. But the exception is the parameter for tiles.
Pyxel's tilemap consists of tiles which is 8x8 pixel sized unit and refers to an image banks. Tiles are represented as a tuple of referencing position (x, y). This (x, y) is the position in tile units. So if their position value change 1, referred image position change 8 pixel.
So I mentioned that (x in tile, y in tile) to notify this number is not pixel-based but tile-based.

Reply:
Ah!  There could be a convention like `tilex, tiley` or `mapx, mapy` to make this clearer.  Or phrases like `tile coordinates` to differentiate from `screen coordinates`.

Reply:
Thank you for your suggestion. I agree. You naming convention is clearer.
I'll replace `x in tile` to `tile_x` in every language's README soon after.
So you can keep them for now.